modules/base/base3d.mjs

```diff
@@ -1,3 +1,7 @@
+///_begin_exclude_in_qt5web_
+import node_canvas from 'canvas';
+import node_gl from 'gl';
+///_end_exclude_in_qt5web_
```

Comment:
This will not work with pure browser. 
It should be as proposed before, hidden by `/* exclude_from_build */`

Reply:
I am exactly trying to figure that out now. I didn't realize this before.

`/* exclude_from_build */` is not going to work here since we import the `mjs` files directly in browsers. It would work if the browser was using `build/jsroot.js`.

I will figure this out. Give me some time.

Reply:
`/* exclude_from_build */` is only matter for bundler.

In normal browser statement ignored while `if (isNodeJs())` statement is false.

Reply:
You always can test jsroot functionality in browser locally with simple python3 macro:

https://github.com/linev/misc/tree/master/server

Or directly from jsroot websites:

https://jsroot.gsi.de/dev/examples.htm